catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -87,7 +87,7 @@
 
         <AD id="onlineGazetteer"
             name="Use Online Gazetteer"
-            description="Should the online gazetteer be used? If unchecked a local one will be used."
+            description="Should the online gazetteer be used? If unchecked a local gazetteer service will be used. This only applies to the search gazetteer in Intrigue."
             type="Boolean"
             default="true"
```

Comment:
Should this default be left to true?

Reply:
This is only for the UI gazetteer so this still should be true. For the UI online gazetteer means using a built in online gazetteer. Not online or local gazetteer means us the ddf gazetteer service which could be and offline or online gazetteer.